user/update.md

```diff
@@ -3,3 +3,36 @@ description: How to update your user's profile and settings.
 ---
 
 # Update user profile and settings
+
+User companyId and roles cannot be updated.
```

Comment:
This is rather a note. Let's first describe where this route is used for, and after that add this note.

Reply:
Added description